### PR TITLE
upgrade dev deps

### PR DESCRIPTION
**Pull request by junnplus**
## What type of PR is this? 

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [x] Other

## How is this tested?

- [ ] Unit tests (pytest, jest)
- [ ] E2E Tests (Cypress)
- [ ] Manually
- [ ] N/A

### CODE REVIEW

File: requirements_dev.txt [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-pytest-cov==2.8.1
-coverage==4.5.4
-mock==3.0.5
-
-# PyMongo and Athena dependencies are needed for some of the unit tests:
-# (this is not perfect and we should resolve this in a different way)
-pymongo[srv,tls]==4.3.3
-boto3>=1.10.0,<1.11.0
-botocore>=1.13,<1.14.0
-PyAthena>=1.5.0,<=1.11.5
```
Comment (junnplus) [inline requirements_dev.txt:11]: During testing, we should include the dependencies of `requirements_all_ds.txt`.

Comment (gaecoli) [reply requirements_dev.txt:11]: When change this file, I don't know if the grass dockerfile needs to be adjusted.

Comment (junnplus) [reply requirements_dev.txt:11]: In #6152, we have made adjustments to the dependencies required for testing.

Comment (gaecoli) [reply requirements_dev.txt:11]: > In #6152, we have made adjustments to the dependencies required for testing.

Cool!

**Pull request merged**